### PR TITLE
dedup --> dedupe

### PR DESCRIPTION
**Pull request by stevage**

### CODE REVIEW

**Review by riastrad — APPROVED** [review]
👍 thanks for keeping our docs legible, @stevage!

**Comment by springmeyer** [comment]
This looks good. But we can't merge it until its passing. To pass it will need to update to latest master after #119. @stevage could you do that?

**Comment by springmeyer** [comment]
Ah, wait, nevermind - looks like I can merge it. Will do that now, as this will pass once merged.

**Pull request merged**
